hips/draw/simple.py

```diff
@@ -178,11 +179,48 @@ def draw_tiles(self) -> np.ndarray:
 
         return image
 
+    def draw_hips_tile_grid(self) -> None:
+        """Draw lines on the output image (mainly used for debugging)."""
+        import matplotlib.pyplot as plt
+        for tile in self.tiles:
+            radec = tile.meta.skycoord_corners
```

Comment:
Yes. That's how I would do it also. This should give you just the lon / lat values in the given frame of the `pos` Skycoord.